waspc/examples/todoApp/src/client/Todo.tsx

```diff
@@ -1,193 +1,216 @@
-import React, { useState, FormEventHandler, ChangeEventHandler } from 'react'
```

Comment:
All these changes are just `Prettier` formatting, safe to skip.

Reply:
The thing is: we doesn't use semicolons, and we use two spaces for indentation. I'm not necessarily a fan of either of these choices, but it's better to stick with conventions :sweat_smile:

Reply:
We should then define a `Prettier` config to make it explicit which conventions we expect. 

I can revert this change, ofc, since it's very aggressive and out-of-place.

Reply:
I'll add the `prettier.config.js` so it's explicit :)